crates/rpc/bridge/Cargo.toml

```
@@ -9,9 +9,15 @@ rust.rust_2018_idioms = { level = "deny", priority = -1 }
 rust.unreachable_pub = "warn"
 rust.unused_crate_dependencies = "deny"
 rust.unused_must_use = "deny"
+rust.missing_docs = "warn"
+rustdoc.all = "warn"
 
 [dependencies]
+strata-bridge-primitives.workspace = true
+
+bitcoin.workspace = true
 jsonrpsee = { workspace = true, features = ["server", "macros", "tokio"] }
```

```suggestion
jsonrpsee = { workspace = true, features = ["server", "macros"] }
```

We don't need `tokio` for the traits, probably in the `dev-dependencies` but it should not go here.